tap_twilio/schemas/cumulative_statistics.json

```diff
@@ -79,91 +79,91 @@
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "max": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "total": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         }
       }
     },
     "wait_duration_until_accepted": {
       "type": [
         "null",
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "min": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "max": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         },
         "total": {
           "type": [
             "null",
-            "number"
+            "integer"
           ]
         }
       }
     },
     "wait_duration_in_queue_until_accepted": {
       "type": [
         "null",
-        "string"
+        "object"
       ],
       "additionalProperties": false,
       "properties": {
         "avg": {
           "type": [
             "null",
-            "number"
+            "integer"
```

Comment:
is this OK with integer? Average usually can be floating point number

Reply:
Their API doc does not define the type of the value, but it does for some which are not embedded into other resources, e.g. `avg_task_acceptance_time`, which is also a whole number.
I did some sample queries to their API, and they do return only whole numbers for these resources.